lang/funcs/operator_polyfunc.go

```diff
@@ -39,6 +39,9 @@ const (
 )
 
 func init() {
+	// epsilon definition
```

Comment:
This is an unhelpful comment. Explain *why* this is the case, since the code is not intuitive.